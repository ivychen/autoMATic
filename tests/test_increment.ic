int main() {
	int i;
	int j;

	i = 0;
	j = i;

	println(i); // 0
  println(j); // 0

  i++;
  j++;

  j = j + 1;
  if (i++ == j) println("i++ == j"); // true
  j = j + 1;
  if (i++ == j) println("i++ == j"); // true
  j = j + 1;
  if (i++ == j) println("i++ == j"); // true

  println(i); // 4
  println(j); // 4

	return 0;
}
