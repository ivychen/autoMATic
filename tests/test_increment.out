0
0
i++ == j
i++ == j
i++ == j
4
4
